File: Limpieza.java
public class Limpieza extends Producto {
    private String marca;

    public Limpieza(String nombre, double precio, String fechaImportacion, int stock, String marca) {
        super(nombre, precio, fechaImportacion, stock);
        this.marca = marca;

    }

    public String obtenerInformacionConMarca() {
        return super.obtenerInformacion() + " Marca: " + marca;
    }

    public void añadirStock(int cantidad) {
        super.agregarStock(cantidad); 
    }

    public String getNombreProducto() {
        return super.getNombre();  
    }
    

    
}

File: Limpieza.py
from Producto import Producto

class Limpieza(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, marca):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.marca = marca

    def obtener_informacion_con_marca(self):
        return f"{super().obtener_informacion()} Marca: {self.marca}"

    def get_nombre_producto(self):
        return super().get_nombre()
